packages/ui/cypress/tests/watched-accounts.cy.ts

```diff
@@ -58,4 +63,154 @@ describe('Watched Accounts', () => {
+      accountDisplay.nameLabel()
+        .should('be.visible')
+        .should('have.text', watchMultisigs['multisig-without-pure'].name)
+    })
+    // ensure the name is included in the selectable drop-down option
+    topMenuItems.multiproxySelector()
+      .should('be.visible')
+      .first().click()
+    topMenuItems.multiproxySelectorOption().within(() => {
+      accountDisplay.nameLabel()
+        .should('have.text', watchMultisigs['multisig-without-pure'].name)
+    })
+    // ensure the name is displayed in the home page header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.nameLabel()
+        .should('have.text', watchMultisigs['multisig-without-pure'].name)
+    })
+  })
+
+  it('can see the provided name displayed for a watched pure', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-with-pure'].pureAddress,
+      watchMultisigs['multisig-with-pure'].name)
+    // ensure the multisig name is displayed in the settings account container
+    settingsPage.accountContainer().within(() => {
+      accountDisplay.nameLabel()
+        .should('be.visible')
+        .should('have.text', watchMultisigs['multisig-with-pure'].name)
+    })
+    // ensure the name is included in the selectable drop-down option
+    topMenuItems.multiproxySelector()
+      .should('be.visible')
+      .first().click()
+    topMenuItems.multiproxySelectorOption().within(() => {
+      accountDisplay.nameLabel()
+        .should('have.text', watchMultisigs['multisig-with-pure'].name)
+    })
+    // navigate to the multisig page and ensure the name is included in the home page header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.nameLabel()
+        .should('have.text', watchMultisigs['multisig-with-pure'].name)
+    })
+  })
+
+  it('can see the identicon and badge displayed for a watched multisig', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-without-pure'].address,
+      watchMultisigs['multisig-without-pure'].name)
+    // ensure the multisig badge is displayed in the selectable drop-down option
+    topMenuItems.multiproxySelector()
+      .should('be.visible')
+      .first().click()
+    topMenuItems.multiproxySelectorOption().within(() => {
+      accountDisplay.identicon().should('be.visible')
+      accountDisplay.multisigBadge().should('be.visible')
+      accountDisplay.pureBadge().should('not.exist')
+    })
+    // navigate to the home page and ensure the multisig badge is included in the header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.identicon().should('be.visible')
+      accountDisplay.multisigBadge().should('be.visible')
+      accountDisplay.pureBadge().should('not.exist')
+    })
+  })
```

Comment:
created the badge-checking tests separate from the name-checking tests, I thought it was best to keep them separate so that if they fail it's immediately clear. Agreed?

Reply:
Ha that was my comment above. To me, I think we could merge them,but I have no strong opinion. I feel like this is bloating minimally the tests, but is very much related to the previous checks.

Reply:
Merged :)